app/services/support_interface/application_references_export.rb

```diff
@@ -24,5 +25,11 @@ def data_for_export
       # we sort by longest hash length to ensure all headers appear
       data_for_export.sort_by(&:length).reverse
     end
+
+    private
+
+    def received_at(reference)
+      reference.audits.where("audited_changes#>>'{feedback_status, 1}' = 'feedback_provided'").last&.created_at
```

Comment:
I've not come across this syntax (`#>>`) before, are you able to explain it to me, please?

Reply:
Hey @raamSoftwire so this was a little tip from David. Basically, we're passing in an sql query based off of the work that Steve did on the candidate_journey_tracker.rb. This check the audit object associated with the reference to find the time of the last audit that occurred when feeback_status was updated to "feedback_provided". This is kind of a work around as 'created_at' does not exist currently as a column in the database. David has added this as a separate card (as we;re now using the workaround in multiple places. (Also, is the below approval for a different PR?)

Reply:
@stevehook  can correct me if I'm wrong, but I believe the functionality used to query JSON blobs can also be applied to hashes. 

It checks nested values. So it looks for audited changes. Which returns this.

`{ "feedback_status" => ["feedback_requested", "feedback_provided"] }`

It then checks that the key is `feedback status` and the value of the second element of the array is `feedback_provided`

https://www.postgresql.org/docs/9.3/functions-json.html